src/NodaTime/Utility/FileLoader.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2011 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using System.IO;
+using System.Net.Http;
+
+namespace NodaTime.Utility
+{
+    /// <summary>
+    /// Provides method to help with loading file or URL.
+    /// </summary>
+    public static class FileLoader
+    {
+        /// <summary>
+        /// Loads a given file or URL, and returns its contents as a byte array.
+        /// </summary>
+        /// <param name="source">Path to the source</param>
+        /// <returns></returns>
+        public static byte[] LoadFileOrUrl(string source)
+        {
+            if (source.StartsWith("http://") || source.StartsWith("https://") || source.StartsWith("ftp://"))
+            {
+                using (var client = new HttpClient())
+                {
+                    // I know using .Result is nasty, but we're in a console app, and nothing is
```

Comment:
And this is the kind of reason I don't want to include it in the public API of the main library :) We don't know we in a console app any more.

Reply:
Can you think of a different implementation?

Reply:
I would rather just not put it into the NodaTime assembly. I'll create a new project as a common one for tools, and use that. As such, it's probably worth closing this PR - but thanks for the nudge.

Reply:
You're welcome. Feel free to close it